hg-fast-export.py

```diff
@@ -122,6 +127,35 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
+lfs_pointer_buffer_size = 1024*1024
+def generate_git_lfs_pointer(filename):
+  size = 0
+  # since Mercurial storage for large files are based around the sha1 we
+  # need to rehash the file to generate the proper key for the git-lfs
+  # pointer file.
+  hasher = hashlib.sha256()
+  with open(filename, "rb") as bfile:
+    buf = bfile.read(lfs_pointer_buffer_size)
+    while buf:
+      hasher.update(buf)
+      size = size + len(buf)
+      buf = bfile.read(lfs_pointer_buffer_size)
+  digest = hasher.hexdigest()
+  try:
+    os.makedirs("lfs/objects/%s/%s" % (digest[0:2],digest[2:4]))
+  except:
+    pass
+  # Once we find where to put the large file, we need to copy it to the
+  # cached location so the git lfs filter replace the file pointer with
+  # large file. If upload is required, it also need to be in the git
```

Comment:
... lfs filter replace the file .. -> ... lfs filter replaces the file...

... it also need to be ... -> ... it also needs to be ...